### PR TITLE
add struct UpdateTeamDeliveryDetailsOptions

### PR DESCRIPTION
**Pull request by worc**

### CODE REVIEW

File: teams.go [hunk 1]
```diff
@@ -27,6 +27,16 @@ type CreateTeamOptions struct {
 	DeliveryRegion   string `json:"delivery_region"`
 }
 
+// UpdateTeamDeliveryDetailsOptions represents the values necessary to update
+// team delivery options, DeliveryLocation: s3 bucket URL, access key / secret key: AWS keys,
+// DeliveryRegion: an AWS region
+type UpdateTeamDeliveryDetailsOptions struct {
```
Comment (dan9186) [inline teams.go]: Looking at the endpoint that actually exists, there is nothing delivery detail specific in the name. We will one that matches the following signature.

```
type UpdateTeamOptions struct {
  ID               string `json:"someid"`
  Name             string `json:"name"`
  POCName          string `json:"poc_name"`
  POCEmail         string `json:"poc_email"`
  DeliveryLocation string `json:"delivery_location"`
  AccessKey        string `json:"access_key"`
  SecretKey        string `json:"secret_key"`
  DeliveryRegion   string `json:"delivery_region"`
}

func (ic *IonClient) UpdateTeam(opts UpdateTeamOptions, token string) (*teams.Team, error) {
  ...
}
```

Comment (worc) [reply teams.go]: yeah, janice is a few commits behind. should there be a function for the delivery update as well?

Comment (dan9186) [reply teams.go]: Per an in office convo. If the goal is to make a separately new endpoint that only updates the delivery options, then we'll want to do `/teams/updateDeliveryDetails` as the endpoint in our service, a struct named `DeliveryDetails`, and a method named `UpdateDeliveryDetails`.

**Comment by matthewkmayer** [comment]
Anything left to do on this PR? Any help wanted for addressing the requested changes?

**Comment by matthewkmayer** [comment]
We'll reopen later if needed. 👍

**Pull request closed without merging**